### DISCUSSION TITLE
How i can add number in HTML?

### DISCUSSION THREAD

**Discussion by quanhoangmanh** [community/community#145377] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**this is test for u can i accept for your medal**

**Comment by tanbaycu** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Add Numbers</title>
</head>
<body>
    <h2>Simple Addition</h2>
    
    <label for="num1">Number 1:</label>
    <input type="number" id="num1" value="0">
    <br><br>

    <label for="num2">Number 2:</label>
    <input type="number" id="num2" value="0">
    <br><br>

    <button onclick="addNumbers()">Add Numbers</button>

    <h3>Result: <span id="result">0</span></h3>

    <script>
        function addNumbers() {
            // Get the values from the input fields
            var num1 = parseFloat(document.getElementById('num1').value);
            var num2 = parseFloat(document.getElementById('num2').value);

            // Add the numbers
            var sum = num1 + num2;

            // Display the result
            document.getElementById('result').textContent = sum;
        }
    </script>
</body>
</html>

**Reply by Ppeepost4489** [reply]
Goog

**Reply by Muhammad-Hasan-Web-Developer** [reply]
?

**Comment by ebndev** [comment]
Hi @tanbaycu, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
